src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        int index = random.nextInt(Colors.values().length);
+        return Colors.values()[index].toString();
```

Comment:
https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#use-name-for-getting-string-representation-of-enum-constants
```suggestion
        return Colors.values()[index].name();
```